app/portainer/views/registries/tags/registryBrowseTagsController.js

```diff
@@ -0,0 +1,119 @@
+angular.module('portainer.app', [])
```

Comment:
Replace this line `angular.module('portainer.app', [])` with `angular.module('portainer.app')`. Otherwise you'll overwrite the `portainer.app` module declaration, causing issues (as such as unable to load the app).